tests/e2e/testdata/scenarios/computefirewallpolicyrule/noupdate/script.yaml

```diff
@@ -38,6 +38,7 @@ spec:
       - "10.100.0.1/32"
   priority: 9000
 ---
+TEST: TOUCH
```

Comment:
Adding `TEST: TOUCH` is excellent. It forces the test runner to reconcile the resource even if the spec hasn't changed, which is exactly what is needed to verify that "no-op" updates don't trigger backend requests.